models/staging/stg_fivetran_platform__transformation_runs.sql

```diff
@@ -0,0 +1,52 @@
+{% if var('fivetran_platform_using_transformations', does_table_exist('transformation_runs')) %}
+
+with base as (
+
+    select * 
+    from {{ var('transformation_runs') }}
+),
+
+fields as (
+
+    select
+        _fivetran_synced,
+        destination_id,
+        upper(free_type) as free_type,
+        job_id,
+        job_name,
+        cast(measured_date as {{ dbt.type_timestamp() }}) as measured_date,
+        model_runs,
+        project_type,
+        updated_at
+    from base
+)
+
+select
+    *,
+    cast({{ dbt.date_trunc('month', 'measured_date') }} as date) as measured_month
+from fields
+
+{% else %}
+
+select
+
+    {% if target.type in ('sqlserver') %}
+    top 0
+    {% endif %}
+
+    cast(null as {{ dbt.type_timestamp() }}) as _fivetran_synced,
+    cast(null as {{ dbt.type_string() }}) as destination_id,
+    cast(null as {{ dbt.type_string() }}) as free_type,
+    cast(null as {{ dbt.type_string() }}) as job_id,
+    cast(null as {{ dbt.type_string() }}) as job_name,
+    cast(null as {{ dbt.type_timestamp() }}) as measured_date,
+    cast(null as {{ dbt.type_int() }}) as model_runs,
+    cast(null as {{ dbt.type_string() }}) as project_type,
+    cast(null as {{ dbt.type_timestamp() }}) as updated_at,
+    cast(null as date) as measured_month
+
+    {% if target.type not in ('sqlserver') %}
+    limit 0
+    {% endif %}
```

Comment:
Do we need to account for Redshift to limit 1 as we've recently uncovered with the union data practices?

cc: @fivetran-jamie

Reply:
Ah that's a great point! I think we should, as we've learned that Redshift won't respect the data type casts otherwise. @fivetran-reneeli can you add a condition to `limit 1` instead of 0 for Redshift targets

Reply:
Yes! I believe I'll need to add it to [usage_cost ](https://github.com/fivetran/dbt_fivetran_log/blob/4ebb0e6d4c1af92960e06c4283323df38d362c85/models/staging/stg_fivetran_platform__usage_cost.sql#L33) and [credits_used](https://github.com/fivetran/dbt_fivetran_log/blob/4ebb0e6d4c1af92960e06c4283323df38d362c85/models/staging/stg_fivetran_platform__credits_used.sql#L33) as well though. I'll add that in and note it in the CHANGELOG.